CHANGELOG.rst

```diff
@@ -4,65 +4,91 @@ Cisco Ios Collection Release Notes
 
 .. contents:: Topics
 
+
+v2.3.0
+======
+
+Minor Changes
+-------------
+
+- Deprecated next_hop_self type bool and introduced nexthop_self as dict under bgp_address_family.
+- Move ios_config idempotent warning message with the task response under `warnings` key if `changed` is `True`
+- PR adds the implementation of object group param to acls source and destination parameters (https://github.com/ansible-collections/cisco.ios/issues/339).
+- PR to fix the bgp global activate rendering and fix bgp address family round trip failure (https://github.com/ansible-collections/cisco.ios/issues/353).
+- To add ospfv2 passive_interfaces param with added functionality (https://github.com/ansible-collections/cisco.ios/issues/336).
+- To add updated prefix lists and route maps params to Bgp AF RM (https://github.com/ansible-collections/cisco.ios/issues/267).
+- To update prefix list and acls merge behaviour and update prefix list description position in model (https://github.com/ansible-collections/cisco.ios/issues/345).
+
+Bugfixes
+--------
+
+- Add support for autoconfig and dhcp keywords for IPv6 addresses in l3_interfaces (https://github.com/ansible-collections/cisco.ios/pull/269).
+- Reordering names of interface for proper value assignment
+- fixes Serial interface configuration for l3_interfaces module and Unit Test cases added.
+- fixes banner module with new attribute introduced
+- fixes soft_reconfiguration and prefix_list command formation.
+
 v2.2.0
 ======
 
 Minor Changes
 -------------
-- Add ios_logging_global module. (https://github.com/ansible-collections/cisco.ios/pull/306).
-- Add Prefix list resource module (https://github.com/ansible-collections/cisco.ios/pull/308).
+
+- Add ios_logging_global module.
```

Comment:
There seems to be a _LOT_ of changes that are not related to this release. Was the antsibull-changelog tool not used for previous releases?

Reply:
@NilashishC I am not sure, maybe it was not used. As the `collection_prep_add_docs` and `antsibull-changelog` produced a  a huge chunk of files